src/platform/forms/tests/save-in-progress/01-sip-load-fail.cypress.spec.js

```diff
@@ -15,29 +15,22 @@ describe('SIP Load Fail Test', () => {
     cy.intercept('GET', '/v1/sessions/new', {
       url: 'http://fake',
     });
-    cy.intercept('GET', '/v0/user', mockUser).as('mockUser');
-    cy.intercept('GET', '/v0/in_progress_forms/1010ez', mock1010Get);
-    cy.intercept('PUT', '/v0/in_progress_forms/1010ez', mock1010Put);
-    cy.login();
+    cy.intercept('GET', '/v0/in_progress_forms/XX-123', mockXX123Get);
+    cy.intercept('PUT', '/v0/in_progress_forms/XX-123', mockXX123Put);
+    cy.login(mockUserSaved);
 
-    cy.visit('/health-care/apply/application');
+    cy.visit('/mock-sip-form');
     cy.get('body').should('be.visible');
-    cy.intercept('GET', '/v0/health_care_applications/enrollment_status', {
```

Comment:
Do we know what this was used for?

Reply:
It is being used specifically in the HCA to showcase whether the logged in user eligible or ineligible to complete the form or if they have a form in progress. Since we are moving away from using the HCA for this testing suite this code has become unneeded and it's removal does not impact the testing suite